### PR TITLE
[KOGITO-1525] Add possibility to set services image tag in Cucumber tests

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-1525

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/framework/kogitodataindex.go [hunk 1]
```diff
@@ -73,18 +69,25 @@ func crInstallKogitoDataIndex(namespace string, replicas int32) error {
...
 	return err
 }
 
+func getDataIndexImage() v1alpha1.Image {
+	if len(GetConfigDataIndexImageTag()) > 0 {
+		return framework.ConvertImageTagToImage(GetConfigDataIndexImageTag())
+	}
+
+	image := framework.ConvertImageTagToImage(infrastructure.DefaultDataIndexImageFullTag)
+	if len(GetConfigServicesImageVersion()) > 0 {
```
Comment (MarianMacik) [inline test/framework/kogitodataindex.go]: There is always a default value for it (version.Version) so I guess we don't need that?

File: test/framework/kogitojobsservice.go [hunk 2]
```diff
@@ -57,17 +57,25 @@ func cliInstallKogitoJobsService(namespace string, replicas int, persistence boo
...
 	return err
 }
 
+func getJobsServiceImageTag() v1alpha1.Image {
+	if len(GetConfigJobsServiceImageTag()) > 0 {
+		return framework.ConvertImageTagToImage(GetConfigJobsServiceImageTag())
+	}
+
+	image := framework.ConvertImageTagToImage(infrastructure.DefaultJobsServiceImageFullTag)
+	if len(GetConfigServicesImageVersion()) > 0 {
```
Comment (MarianMacik) [inline test/framework/kogitojobsservice.go]: Same here.

**Comment by radtriste** [comment]
added bats tests

File: test/framework/kogitodataindex.go (hunk 1 above)
Comment (radtriste) [reply test/framework/kogitodataindex.go]: updated

File: test/framework/kogitojobsservice.go (hunk 2 above)
Comment (radtriste) [reply test/framework/kogitojobsservice.go]: updated

**Pull request merged**